spec/javascripts/reducers/project_spec.js

```diff
@@ -0,0 +1,36 @@
+import actionsTypes from "../../../app/assets/javascripts/actions/actionTypes";
+import reducer from "../../../app/assets/javascripts/reducers/project";
+
+describe("Project Reducer",() => {
+  describe("RECEIVE_PROJECT",() => {
+    it("returns action.data",() => {
+      const state = {};
+
+      const action = {
+        type: actionsTypes.RECEIVE_PROJECT,
+        data: {id: 1}
+      };
+
+      expect(reducer(state, action)).toEqual(action.data);
+    });
+  });
+
+  describe("ADD_LABEL_TO_PROJECT",() => {
+    it("adds new label to labels",() => {
+      const state = { labels: []};
+
+      const action = {
+        type: actionsTypes.ADD_LABEL_TO_PROJECT,
+        label: {name: "label"}
+      };
+
+      expect(reducer(state, action)).toEqual({ labels: [{  name: "label"}]});
```

Comment:
I don't know much about the project code style, but it looks a bit weird. In case of suggestion is wrong, adjust to match the style.
```suggestion
      expect(reducer(state, action)).toEqual({ labels: [{ name: "label" }] });
```